### ISSUE TITLE
Re0 randomizer stopped executing

### ISSUE THREAD

**Issue by Chipper366** [HansRandomizer/Resident-Evil-0-Randomizer#8]
Hi, love this randomiser and hope your development keeps improving.
I have experienced the randomiser executable refusing to run at times after a crash and once it happens never been able to run it again.
I seem to have found out what fixes it:
If you go to display settings and drop scaling setting from recommended (300% on 4K) to a lower number then it activates the exe every time.
Hope this helps you with a fix going forward.

**Comment by HansRandomizer** [comment]
Hi there, thank you for reporting this! Do you have this issue with the configuration tool or with the actual randomizer? @Chipper366

**Comment by Chipper366** [comment]
The problem is only with randomised.exe
I have the same problem with 2 PC's one 4k and one 1080p
Both have old Radeon RX 580  graphic cards in them, wonder if it's a problem with AMD drivers.

**Comment by HansRandomizer** [comment]
The issue arises only after a crash? What is exactly crashing? Sometimes, Cheat Engine doesn't terminate as it should. Could you verify with Task Manager if there are any instances of 'randomizer.exe' or 'cheatengine.exe' running in the background? If you find any, end them via Task Manager and then give it another try.

Also, be aware that there was a bug with the balanced mode, but it has been addressed in the 1.1.1 release candidate. This should eliminate crashes during Item pickup. 

Additionally, it would be helpful if you could create a new issue regarding the crashes you've encountered. Kindly provide logs and the necessary details to reproduce these issues.

**Comment by Chipper366** [comment]
Hi Hans,
Sorry I may have steered you in the wrong direction with mentioning "after a crash"
PC 1 Intel CPU RX 580 1080P screen
PC 2 AMD CPU RX 580 4K screen

PC 1 was my main use and initially no problems starting or restarting a random seed. Until 1 hard crash then nothing would allow me to reopen the rando.exe
Tried uninstalling the game graphics card drivers etc.

PC 2 download rando for the first time, I think it was v1.1.0 and the rando.exe wouldn't execute at all nothing running in task manager at all.

The only way I can get both PC's to work is to change the display scaling then works instantly on both PC's and i am able to reproduce the non functioning rando.exe by changing the scaling settings back to the recommended settings.
I hope this makes sense and it may be my machine's specific setups BTW PC 1Windows 10 PC 2 Windows 11
When I was researching my issues I did come across another post elsewhere with the same problem I think
